core/shared/src/main/scala/scalaz/zio/IO.scala

```diff
@@ -788,6 +790,13 @@ object IO {
       )
     )
 
+  final def shift[E, A](ec: ExecutionContext): IO[Nothing, Unit] =
+    IO.async { cb: Callback[Nothing, Unit] =>
+      ec.execute(new Runnable {
```

Comment:
This belongs in `RTS` where an `ExecutionContext` will be provided.

Reply:
Not that it's a big deal, but are you essentially indicating that `zio` does not plan to support pool shifting outside of the one defined in the `RTS`?

Reply:
don't know the plans save for [this](https://github.com/scalaz/scalaz-zio/issues/36), I'm just a happy contributor 🤗

Reply:
As far as I understood @jdegoes, this is just a stepping stone to the #36, so I think we're all on the same page :).